js/integration-tests/solidity-tests/test/testContext.ts

```diff
@@ -0,0 +1,98 @@
+import {
+  Artifact,
+  ArtifactId,
+  FuzzConfigArgs,
+  InvariantConfigArgs,
+  type SolidityTestRunnerConfigArgs,
+} from "@nomicfoundation/edr";
+import {
+  buildSolidityTestsInput,
+  runAllSolidityTests,
+} from "@nomicfoundation/edr-helpers";
+import hre from "hardhat";
+
+export class TestContext {
+  readonly rpcUrl = process.env.ALCHEMY_URL;
+  readonly rpcCachePath: string = "./edr-cache";
+  readonly fuzzFailuresPersistDir: string = "./edr-cache/fuzz";
+  readonly invariantFailuresPersistDir: string = "./edr-cache/invariant";
+  readonly artifacts: Artifact[];
+  readonly testSuiteIds: ArtifactId[];
+
+  constructor(artifacts: Artifact[], testSuiteIds: ArtifactId[]) {
```

Comment:
Normally a static async constructor means a private constructor, unless you want to allow creating instances directly too.

Reply:
Thanks fixed in [4f96f26](https://github.com/NomicFoundation/edr/pull/652/commits/4f96f26ee817db63b623f52fa5baf9d0efc4c73c)